docs/en/docs/network/performance/cni-performance.md

```diff
@@ -13,43 +13,43 @@ This test report covers the performance testing of CNIs including Cilium, Calico
 
 ### Test environment
 
-|Model|CPU|Memory|NIC|System|Node IP|
-|--|--|--|--|--|--|
-|Dell PowerEdge R620|Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz * 2|4G 1333M * 4|Mellanox cx5 Physical NIC Bandwidth|Ubuntu 22.04|10.20.0.11|
-|Dell PowerEdge R620|Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz * 2|4G 1333M * 4|Mellanox cx5|Ubuntu 22.04|10.20.0.12|
+| Models | CPUs | Memories | NICs | Systems | Node IPs |
+| --- | --- | ---| --- | --- | --- |
+| Dell PowerEdge R620 | Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz * 2 | 4G 1333M * 4 | Mellanox cx5 Physical NIC Bandwidth | Ubuntu 22.04 | 10.20.0.11 |
+| Dell PowerEdge R620 | Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz * 2 | 4G 1333M * 4 | Mellanox cx5 | Ubuntu 22.04 | 10.20.0.12 |
 
 ### Tested CNI types
 
-This test covers three CNIs including Cilium, Calico and Macvlanis and is based on the following models:
+This test covers three CNIs including Cilium, Calico and Macvlan, and is based on the following models:
 
 - Cilium
-    - cilium (vxlan mode)
+    - Cilium (VXLAN mode)
 - Calico
-    - calico (vxlan mode)
-    - calico (underlay mode)
-    - callico (ipip mode)
+    - Calico (VXLAN mode)
+    - Calico (underlay mode)
+    - Calico (IPIP mode)
 - Macvlan-Standalone
 
 ### Test cases
 
 This test includes the following nine Test cases:
 
 - Node to node
-- Pod to node where the Pod resides
+- Pod to node where the pod resides
 - Pod to node across nodes
-- Pod to Pod on the same node
-- Pod to Pod across nodes
+- Pod to pod on the same node
+- Pod to pod across nodes
```

Comment:
Need to use a consistent style with the following tables:

- on the same node
- across nodes